build-vm-nspawn

```diff
@@ -0,0 +1,74 @@
+#
+# Docker specific functions
+#
+################################################################
+#
+# Copyright (c) 2019 Oleg Girko
+# Copyright (c) 2019 SUSE Linux Products GmbH
```

Comment:
Don't put SUSE copyright unless there's a reason to...

Reply:
OK, removed.